### PR TITLE
Release 4.0.0

### PR DESCRIPTION
**Pull request by FinalAngel**
**This pull request stays open until proper release:
https://github.com/aldryn/aldryn-boilerplate-bootstrap3/blob/feature/4.0.0/boilerplate.json#L4**

For review purpose, ignore changes made to the ``private/sass/libs/`` directory.

4.0.0
=====
- added libsass support
- updated ``templates/404.html`` to look better
- updated Font Awesome to 4.4.0
- updated Font Awesome to use sass library
- updated content templates
- updated license information
- updated documentation
- removed compass support
- use `@error` instead of `@warn` in sass
- removed `lh` from ``mixins/_functions.scss``

### CODE REVIEW

File: tests/integration/specs/spec.dummy.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
-/*!
- * @author:    Divio AG
- * @copyright: http://www.divio.ch
+/*
+ * Copyright (c) 2013, Divio AG
```
Comment (vxsx) [inline tests/integration/specs/spec.dummy.js]: 2015?

File: LICENSE.txt [hunk 2]
```diff
@@ -1,33 +1,24 @@
-# Aldryn Boilerplate Standard
-# Copyright Divio AG
-# https://github.com/divio
-
-Some rights reserved.
+Copyright (c) 2013, Divio AG
```
Comment (lory87) [inline LICENSE.txt]: Update year

File: gulpfile.js [hunk 3]
```diff
@@ -90,10 +95,36 @@ gulp.task('lint:sass', function () {
...
 // #############################################################################
 // PREPROCESSING
-gulp.task('preprocess', ['images', 'docs']);
+gulp.task('preprocess', ['sass', 'images', 'docs']);
+
+gulp.task('sass', function () {
+    gulp.src(PROJECT_PATTERNS.sass)
+        // sourcemaps can be activated through `gulp sass --debug´
+        .pipe(gulpif(DEBUG, sourcemaps.init()))
+        .pipe(sass())
```
Comment (lory87) [inline gulpfile.js]: Is it possible to always compress the outputted css ?
{ outputStyle: 'compressed' }

Comment (lory87) [inline gulpfile.js]: hmm sorry saw you added minifyCss

File: private/sass/base.scss [hunk 4]
```diff
@@ -1,7 +1,8 @@
 @charset "utf-8";
-/*!
- * @author:    Divio AG
- * @copyright: http://www.divio.ch
+/*
+ * Copyright (c) 2013, Divio AG
```
Comment (lory87) [inline private/sass/base.scss]: correct year?

**Comment by lory87** [comment]
:+1:

File: tests/integration/specs/spec.dummy.js (hunk 1 above)
Comment (FinalAngel) [inline tests/integration/specs/spec.dummy.js]: the copyright statement stands for the year it has been licensed to, which is 2015

File: gulpfile.js (hunk 3 above)
Comment (FinalAngel) [inline gulpfile.js]: np :)

**Comment by Brankub** [comment]
Awesome :+1:

**Comment by vxsx** [comment]
6821131-titanic-wallpaper

**Comment by FinalAngel** [comment]
:D

**Pull request merged**